test/rsrc/convert_stub.py

```diff
@@ -20,8 +27,11 @@ def convert(in_file, out_file, tag):
     # On Windows, use Unicode paths. (The test harness gives them to us
     # as UTF-8 bytes.)
     if platform.system() == 'Windows':
-        in_file = in_file.decode('utf8')
-        out_file = out_file.decode('utf8')
+        if not six.PY2:
+            in_file = in_file.encode(arg_encoding())
+            out_file = out_file.encode(arg_encoding())
+        in_file = in_file.decode('utf-8')
+        out_file = out_file.decode('utf-8')
```

Comment:
You can see it here: https://ci.appveyor.com/project/beetbox/beets/build/824/job/226mw7hfg9vdrkto#L64